project_script/main.py

```diff
@@ -77,6 +73,19 @@ def ask_y_axys():
 
 data_ramp_time = pd.concat([time_photo_celiv_ramp_time, even_columns_subtraction], axis=1).sort_index(1, 1)
 
-# data_ramp_time.to_csv(ask_file_name(), sep='\t')
+data_ramp_time.to_csv(ask_file_name(), sep='\t')
+
+data_ramp_time_transposed_in_arrays = data_ramp_time.transpose().to_numpy()
+
+
+integrated_val = integrate\
+    .simps(data_ramp_time_transposed_in_arrays[3], data_ramp_time_transposed_in_arrays[2], axis=-1, even='avg')
```

Comment:
Suggestion. You could break just de args, like this, but i don't know if it's possible in python sintax, it's more readable. Check pls.

```
integrated_val = integrate.simps(\
    data_ramp_time_transposed_in_arrays[3],\
    data_ramp_time_transposed_in_arrays[2],\
    axis=-1, even='avg'\
)
```